src/main/java/seedu/mark/model/reminder/Reminder.java

```diff
@@ -0,0 +1,58 @@
+package seedu.mark.model.reminder;
+
+import static seedu.mark.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Date;
+import java.util.Objects;
+
+import seedu.mark.model.bookmark.Bookmark;
+
+
+/**
+ * Represent a reminder in Mark.
+ */
+public class Reminder {
+    //date field
+    private Bookmark bookmark;
+    private Date remindTime;
+
+    //identity field
+    private Note note;
+
+    /**
+     * Every field must be present and not null.
```

Comment:
Perhaps you can add `@params` to the the javadoc comment?

Reply:
Okay.